client/src/App.js

```diff
@@ -113,7 +145,11 @@ class App extends Component {
                   Login
                 </Button>
               )}
-
+              {this.state.currentUser && <ListItemAvatar>
+                <Avatar
```

Comment:
This is lookin good, but the avatar should be removed in the logout() function when the user clicks logout from the home page. We just fixed this issue (not in this PR, but coming) by changing initial currentUser to null.